docs/api-reference/view.md

```diff
@@ -53,6 +53,12 @@ If `projectionMatrix` is not supplied, `Viewport` will build a matrix from the f
 * `focalDistance`=`1` (`Number`) - (orthographic projections only) The distance at which the field-of-view frustum is sampled to extract the extents of the view box. Note: lso used for pixel scale identity distance above.
 * `orthographicFocalDistance` (`Number`) - (orthographic projections only) Can be used to specify different values for pixel scale focal distance and orthographic focal distance.
 
+* `controller` (`Function` | `Boolean` | `Object`) - options for viewport interactivity.
+  - If falsy, this view is not interactive.
+  - If set to `true`, the default controller options are used.
```

Comment:
I'd suggest reworking along these lines:
```
- `true`: instantiates the default controller class with default options.
- `Controller` class (not instance): instantiates the provided controller class with default options
```

Calling it a "constructor" is technically correct but I think we prefer to think of it as a "class".